cypress/integration/work-order/authorisation/variation-authorisation.spec.js

```diff
@@ -360,4 +360,72 @@ describe('Contract manager can authorise variation', () => {
       cy.contains('td', '£10')
     })
   })
+
+  it('Can reject variation for Purdy contractor wirh unlimited characters for notes and will show summary page before submitting rejection', () => {
```

Comment:
We can rework this a bit to avoid repeating the long string. At the moment it's tricky to see if the string changes or if it's just the same thing over and over.

We can define the var like this with the relevant string length passed into `.repeat()` - also avoids the need for any specific copy and surfaces the length which is the important bit - also means we don't need a comment about the length.

```
  cy.wrap('x'.repeat(251)).as('longString')
```

Then add a callback and place everything that needs to refer to the string within it:

```
    cy.get('@longString').then((longString) => {
      cy.get('#note').type(longString)
      // other commands
      cy.contains(`This work is too complicated: ${longString}`)  // interpolated into another string
      // other commands
    })
```